aws/cloudwatch-logs-delivery/main.tf

```diff
@@ -0,0 +1,118 @@
+locals {
+  function_name = "${var.name}cloudwatch-logs-delivery-${random_id.unique_id.dec}"
+}
+
+resource "aws_lambda_function" "cloudwatch_log_sentry_notification" {
```

Comment:
Is there any common code for the Sentry destinations that we could extract to a module? At a glance, it looks like most of the resources for the Lambda function are the same between the different sources, but maybe the Python code is a little different in each?